app/models/concerns/direct_db_conn.rb

```diff
@@ -0,0 +1,65 @@
+module DirectDBConn
```

Comment:
True, it's not really a concern. I just didn't want to put it into the models directory since i didn't want a module there. I can change here!

About the table on the instance, i was just thinking that this module could be used to more things (even on the same model, for different tables). But i guess then the person could just create more instances to use it... 

Good call on this! Will change here.